src/AuthenticationPortal.Web/Validations/SignInRequestValidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using FluentValidation;
+
+namespace AuthenticationPortal.Web
+{
+    public class SignInRequestValidator : AbstractValidator<Contracts.SignInRequest>
+    {
+        public SignInRequestValidator()
+        {
+            RuleFor(x => x.Username)
+            .NotNull()
```

Comment:
please discuss with Abhishek regarding validations ,please go through he changes suggested to him if applicable

Reply:
Done